test/unit/three_scale/redis_config_test.rb

```diff
@@ -43,5 +43,23 @@ class RedisConfigTest < ActiveSupport::TestCase
       assert config.key? :size
       assert_equal 5, config[:size]
     end
+
+    test "it takes given ca_file when provided" do
+      value = 'any_value'
+      raw_config = { url: 'rediss://my-secure-redis/1', ssl_params: {}}
+      raw_config[:ssl_params][:ca_file] = value
+
+      result = RedisConfig.new(raw_config)
+
+      assert result.key? :ssl_params
+      assert result[:ssl_params].key? :ca_file
+      assert_equal value, result[:ssl_params][:ca_file]
+    end
+
+    test "it doesn't trust any CA if no ca_file is provided" do
```

Comment:
```suggestion
    test "it doesn't set CA if no ca_file is provided in config" do
```

Reply:
Right, it'll trust the SO cert bundle in this situation